### ISSUE TITLE
microG - Proposal

### ISSUE THREAD

**Issue by MSe1969** [derfelot/discussion#16]
The two listed changes (rejected by CyanogenMod) are a port of the OmniROM Custom ROM. They combine the patch I use to build my microG-patched version (btw, also rejected by CM) with the option to enable/disable that feature in the developer settings.

https://review.lineageos.org/#/c/67174/
https://review.lineageos.org/#/c/130219/

So by default, sig-spoofing is disabled/not available, until a user explicitly switches it on in the developer settings. This would allow us to have one common build.

What do you think?

**Comment by Micha-Btz** [comment]
For me ok. I have had a test with microg in the last week and the device have had some strange reboots.
But when it is not enabled by default it is fine.

**Comment by MSe1969** [comment]
Thanks for the feedback, so I'll push them the two commits (I've already tested successfully).

microG runs perfectly stable on all my devices - so just to make sure:
You have maybe seen in the "welcome" issue, that I've missed to merge the kernel from cm-12.1-test to cm-12.1, when I was merging all other stuff (I did meanwhile), and the test build with the new microG-feature caused my device because of that to bootloop (because of enabled encryption, and not because of mircoG). After fixing the kernel merge, all works fine.

If you had a different issue with microG, just send me a PM in XDA, I do have some experience with it.
